packages/font-icons-mdl2/src/index.ts

```diff
@@ -87,7 +87,9 @@ export function initializeIcons(
   registerIconAliases();
 }
 
-// eslint-disable-next-line deprecation/deprecation
-export { IconNames, IconNamesInput } from './IconNames';
+/* eslint-disable deprecation/deprecation */
+export type { IconNamesInput } from './IconNames';
```

Comment:
although this package is the only one in monorepo that cannot use isolateModules because const enum is used ( it would increase customer bundle size if they use typescript for compilation. if customer is not using typescript this wouldn;t be breaking change) we still get warnings from swc/babel because incorrectly re-exported type.